install/pre-install-payload/Dockerfile

```diff
@@ -69,10 +69,9 @@ ENV GOARCH=${ARCH}
 
 RUN mkdir -p ${NODE_DESTINATION}/bin && \
     apk add --no-cache curl && \
-    if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
-    curl -fOL --progress-bar ${NYDUS_SNAPSHOTTER_REPO}/releases/download/${NYDUS_SNAPSHOTTER_VERSION}/nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz && \
-    tar xvzpf nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz -C / && \
-    rm nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz && \
+    curl -fOL --progress-bar ${NYDUS_SNAPSHOTTER_REPO}/releases/download/${NYDUS_SNAPSHOTTER_VERSION}/nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz && \
+    tar xvzpf nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz -C / && \
+    rm nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz && \
     mv /nydus-snapshotter/* ${NODE_DESTINATION}/bin/ && \
```

Comment:
The v0.13.11 doesn't use the `nydus-snapshotter` prefix inside the tarball so you need to remove the lines `75,76` (mv and rm /nydus-snapshotter)

Reply:
This is a good catch. I think I made a slightly different but appropriate change and see enclave-cc test passing now